03_oop_part_2/src/page.ts

```diff
@@ -0,0 +1,15 @@
+export class Page {
+    private pageNumber: number;
+    private pageType: string;
```

Comment:
@romasaldan, it is better to use enum, because property has defined set of values ("with text", "with article", "with images")